lib/brain.js

```diff
@@ -690,6 +690,9 @@ Brain.prototype.hasExpired = function hasExpired (cardData) {
   // TODO: ZA cards currently do not have an expiration date to confirm against
   if (cardData.country === 'ZA') return false
 
+  // In case the expiration date field is not found. Prevents l-m from bailing
+  if (_.isNil(cardData.expirationDate)) return false
```

Comment:
Can't a user just cover up their expiration date to make this succeed? Maybe in this case it should return ``true``.

Reply:
You're correct, seems obvious the edge case should be treated that way, now that I think about it